torchtune/models/qwen2/_convert_weights.py

```diff
@@ -68,7 +68,7 @@ def qwen2_hf_to_tune(
 
     for key, value in state_dict.items():
         if (
-            tie_word_embeddings and QWEN2_TIED_KEY not in key
+            tie_word_embeddings and QWEN2_TIED_KEY in key
```

Comment:
I missed this the first time, but is there a reason the checkpointing logic for tied embeddings in Qwen2 should be different from Gemma? Ref #1168, #1169

Reply:
@ebsmothers @joecummings

Reply:
Good point, happy to have this filed as a refactor cleanup to start grouping some of these together in a "TIED_MODEL" key or something.

Reply:
Sorry if I'm missing something, but do we need a refactor here? I thought we ended up not needing any special checkpointing logic for Gemma, so it's just a matter of applying the same changes as in https://github.com/pytorch/torchtune/pull/1168/files - removing `qwen2/_convert_weights.py`, and remove any special logic in the checkpointers ([here](https://github.com/pytorch/torchtune/blob/72ef1ddac2d3d155b54e1c8323a6d865ba0bd0ee/torchtune/utils/_checkpointing/_checkpointer.py#L451), for example), so it goes through the default model save/load path.